src/icons/annotations/IconRegionAnnotation.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { white } from '../../styles/variables';
+
+import AccessibleSVG from '../accessible-svg';
+
+import { Icon } from '../iconTypes';
+
+const IconRegionAnnotation = ({ className = '', color = white, height = 21, title, width = 22 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-annotation-region ${className}`}
```

Comment:
Should `icon-annotation-region` -> `icon-region-annotation` to be consistent?

Reply:
I followed the other existing classnames

Reply:
I realized we may also want to do SUIT styles, `bdl-IconRegionAnnotation`